iscsi/targetd/Makefile

```diff
@@ -0,0 +1,52 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+IMAGE = raffaelespazzoli/iscsi-controller
+
+VERSION :=
```

Comment:
we've mostly converged on a new makefile format: https://github.com/kubernetes-incubator/external-storage/blob/master/gluster/block/Makefile

the TAG & VERSION logic here won't work anymore since we have all sorts of tags now like "v1.0.0" "efs-provisioner-v0.1.0", etc.